app/models/header.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+
+const Header = function (data, db) {
+  this.data = data;
+  this.db = db;
+};
+
+Header.prototype.data = {};
```

Comment:
@rodrigopavezi I may have missed it, but I don't think this shared property is used anywhere. Can it be removed?